---
permalink: /
title: "Welcome! Xiavan here."
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

I am a Ph.D candidate in Biochemistry and Molecular Biology at the [University of Oklahoma Health Sciences Center (OUHSC)](https://basicsciences.ouhsc.edu/bmb/) and I do my research in [Jonathan Wren's lab at the Oklahoma Medical Research Foundation (OMRF)](https://omrf.org/research-faculty/scientists/wren-jonathan-d/). My thesis research leverages the influx of avaiable DNA methylation microarray data to identify how gene function is associated with the methylation status of a given gene. Using traditional machine learning as well as deep-learning techniques, I generate predictions for the relationship between genes and evaluate how predictive different regions of the methylome are compared to the existing methods using gene expression. 


Background
======
I received my B.S. in Biology-Chemistry in 2013 from Southern Nazarene University in Oklahoma. After that, I was a research technician at OMRF where I built my skills as a programmer and developed my research interests in informatics and exploring the biomedical data available in repositories like the [Gene Expression Omnibus](https://www.ncbi.nlm.nih.gov/geo/) and [Sequence Read Archive](https://www.ncbi.nlm.nih.gov/sra). I developed tools to help prepare and analyze collaborator sequencing and microarray data, as well as built and administered the hardware and software necessary for high performance Linux computing in my lab. In grad school, I rotated through labs to develop a solid foundation in wet lab techniques including cell culture and high-throughput sequencing. 

Interests
======
Using cutting-edge techniques to develop models leveraging biomedial data. I enjoy developing useful presentations that communicate complex findings catered to the audience's domain knowledge, as well as adapting new and fresh data visualizations to enhance the way I present findings. As part of the skillset required for effective bioinformatics work, I am proficient and always learning more about managing software and Linux servers, as well as cluster computing and parallelization tools. Similarly, deep learning and GPU computing has become an increasingly important part of the way I approach problems, and I am always happy to discuss developments in neural networks, whether it's the latest OpenAI paper, or clinical implementations of deep learning models as diagnostic tools.  

Skills
======
1. **Next-gen sequencing** QC, alignment, differential expression analysis, as well as downstream pathway and GO analysis. Experienced with analyzing data from Illumina platforms
1. **DNA methylation analysis** - Analyzing Illumina methylation microarrays and bisulfite sequencing data, performing meta-analysis and differential methylation
1. **Data visualization** - Proficient with visualization libraries in Python, R, Javascript and Julia - especially Seaborn and Plotly
1. **Linux skills** - I'm comfortable with managing and troubleshooting the Linux-based software required for cutting edge machine learning and data analysis
1. **Statistics** - Knowledgable of proper controls, testing, and statistical corrections essential for evaluating results, and are necessary considerations for experimental design and sequencing runs
1. **Machine learning** - I frequently utilized traditional machine learning to predict a feature of interest for collaborator analyses, and am proficient with Python and R libraries for machine learning including scikit-learn and limma. 
1. **Deep learning** - I have leveraged both PyTorch and Keras libraries in Python for predicting gene function using DNA methylation, and am generally familiar with the tools necessary for training a neural network. 
