---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* B.S. in Biology-Chemistry, Souther Nazarene University, 2013
* Ph.D in Biochemistry, University of Oklahoma Health Sciences Center, 2021 (expected)

Work experience
======
* Summer 2013-2015: Research Technician
  * Oklahoma Medical Research Foundation
  * Duties included: profiling simple tandem repeats in human genomes using lobSTR, profile DNA methylation across microarrays available in GEO
  * Supervisor: Jonathan Wren

* Fall 2012: Research Assistant
  * Southern Nazarene University
  * Duties included: establishing standard curve for cholesterol esterase activity
  * Supervisor: Daryl Cox

* Fall 2010-2012: Teaching Assistant
  * Southern Nazarene University
  * Duties included: Supervised, lectured, and prepared tests for labs in Cells and Systems
  * Supervisor: Caroline Bentley

Skills
======
* Python. Seaborn, Pandas, Numpy, Scipy, Scikit-learn, Statsmodels, Plotly, Dash
* R. ggplot, Limma, Methylumi, deseq2
* Keras and PyTorch
* Linux and Cluster computing
* Next-Gen Sequencing alignment and analysis, Bismark, Bowtie, Sam/Bamtools, FastQC, Kallisto, Arioc
* DNA methylation analysis
* Julia

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
Talks
======
  <ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>

Conference Presentations
======
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. BMB Departmental Seminar Series, OUHSC, 2017
* Collapsing DNA Methylation to Informative Genomic Features for Biological Classification. BMB Departmental Seminar Series, OUHSC, 2018
* Collapsing DNA Methylation to Informative Genomic Features for Biological Classification. BMB Departmental Seminar Series, OUHSC, 2019
* DNA methylation’s predictive capacity for biological annotations is dispersed across the methylome and not limited to promoter regions. BMB Departmental Seminar Series, OUHSC, 2020
* Predicting gene function from a global methylation microarray network meta-analysis. BMB Departmental Seminar Series, OUHSC, 2021
* Introduction to Data Visualization, Scientific Python User’s Group (SPUG), OUHSC, 2019
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. Genes and Human Disease Department Seminar, OMRF, 2020
* Biological Classification Using Collapsed Feature Methylation. GREAT, OUHSC, 2016
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. GREAT, OUHSC, 2017
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. GREAT, OUHSC, 2018
* Assessing Correlation of Methylation State of CpG Sites within CpG Islands. MCBIOS, OUHSC, 2016
* Assessing Correlation of Methylation State of CpG Sites within Promoter Regions. MCBIOS, OUHSC, 2017
* Gene Function Prediction Using Global Meta-Analysis of Promoter-Associated CpG Comethylation. MCBIOS, OUHSC, 2018
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. OU BMB Retreat, 2017
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. OMRF Retreat, 2017-18, OMRF
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions. Oklahoma Geroscience Symposium, 2018-2019
* American Aging Association (AGE) 48th Annual Meeting, 2019
* DNA methylation’s predictive capacity for biological annotations is dispersed across the methylome and not limited to promoter regions. ISMB, Virtual Conference, 2020
* DNA methylation’s predictive capacity for biological annotations is not limited to promoter regions

