---
permalink: /experience/
title: "Research Experience"
author_profile: true
---

## Prediction of gene function using DNA methylation
Developed the first published gene function prediction model using DNA methylation. Identified increased predictive performance of gene body methylation compared to all regions of the methylome, and increased predictive performance of integrated promoter methylation and expression methylation compared to individual models trained on a single data type.


Xiavan‌ ‌Roopnarinesingh, ‌Hunter‌ ‌Porter,‌ ‌Cory‌ ‌Giles,‌ ‌Chase‌ ‌Brown,‌ ‌Constantin‌ ‌Georgescu,‌ ‌Jonathan‌ D. ‌Wren‌. Manuscript to be submitted. “Multi-tissue DNA methylation microarray signature is predictive of gene function”

## Correlation to expression across the methylome

Correlated different regions of the methylome to expression in paired methylation data from TCGA. These genes were compared to the gene function prediction performance, with gene body prediction performance increasing with correlation to expression, while the predictive performance decreased with correlation to expression. Identified varied associations of methylation expression including repressive hypermethylation of promoters and hypermethylation in gene bodies across varied expression levels. 


Code repository with tools available on GitHub: [https://github.com/xroopnar/GDC-paired-expression](https://github.com/xroopnar/GDC-paired-expression)

## Trend deviation analysis

By combining the global correlation network of DNA methylation microarray data in the Gene Expression Omnibus, I created a tool to compare the correlation structure of a given experiment’s differentially methylated genes to the global signature and identify changes (trend deviation). This has been applied to collaborator data on lupus to identify potentially disease-related genes. 


Manuscript in progress

## Scientific computing and deep learning 

Managed Linux distributions for shared computing infrastructure, including Bismark bisulfite sequencing workflows, Cellranger, and GPU-accelerated Tensorflow, as well as serving data from disk arrays over network for collaborator use. Contributed to developing tools for automatically annotating GEO data with annotation information. Developed workshops for scientists with novice to intermediate programming skills to make plots in Plotly and Seaborn and interactive webapps in Python Dash using microarray and sequencing data. Developed and presented deep learning autoencoders for DNA methylation gene function prediction. 


Cory B. Giles, Chase A. Brown, Michael Ripperger, Zane Dennis, Xiavan Roopnarinesingh, Hunter Porter, Aleksandra Perz, and Jonathan D. Wren. ALE: automated label extraction from GEO metadata. BMC Bioinformatics. 2017;18(S14):509.


Recorded presentations for Data Science Workshop: [https://xroopnar.github.io/talks/](https://xroopnar.github.io/talks/)
